search/views.py

```diff
@@ -99,13 +106,13 @@ def calculator(request):
 def is_file_extension_ignored(file_):
     return file_.split('.')[-1] in ['md', 'MD']
 
-
-# Search query
 def query(request):
     global algo_name, title
     if request.method == 'GET':
         query = re.escape(request.GET['q']).replace('\ ', ' ')
-
+		#add
```

Comment:
Please remove comments that such this, since the `diff` will highlight those new code line or modified one.